_platforms/en/io.md

```diff
@@ -0,0 +1,58 @@
+---
+title: IO
+subtitle: The app of the public services
+logo: /assets/images/logo-io@2x.png
+external_website: https://io.italia.it
+payoff: The app of the public services
+description: io is the app of the Italian Government that will allow to all the citizens to use the national and local public services from their own smartphone in a simple, modern and secure way.
+owners:
+    - name: PagoPA S.p.A.
+      url: https://www.pagopa.gov.it/
+lang: en
```

Comment:
```suggestion
lang: en
ref:
  it: /it/io
```

We need this to link to the Italian page in the language switcher
(also, we need `en: /en/io` in the Italian page, to do the opposite)